deployment/deploy_parameters.json.example

```diff
@@ -14,9 +14,9 @@
     "timelockAddress": "0x617b3a3528F9cDd6630fd3301B9c8911F7Bf063D",
     "minDelayTimelock": 3600,
     "salt": "0x0000000000000000000000000000000000000000000000000000000000000000",
-    "zkEVMDeployerAddress":"",
     "initialZkEVMDeployerOwner" :"0xaddress"
```

Comment:
add `,`

Reply:
What you mean by add ,

Reply:
simply add a comma at the end of line to make it a valid json file.